src/content/docs/en/guides/internationalization.mdx

```diff
@@ -0,0 +1,325 @@
+
+```
+├── src
+│   ├── pages
+│   │   ├── en
+│   │   │   ├── about.astro
+│   │   │   ├── index.astro
+│   │   ├── es
+│   │   │   ├── about.astro
+│   │   │   ├── index.astro
+│   │   ├── pt_BR
+│   │   │   ├── about.astro
+│   │   │   ├── index.astro
+│   ├── index.astro
+```
+
+Compute relative URLs for your links with `getLocaleRelativeURL` from the new `astro:i18n` module:
+
+```astro
+---
+import {getLocaleRelativeUrl} from "astro:i18n";
+const aboutUrl = getLocaleRelativeUrl("pt_Br", "about");
+---
+<p>Learn more <a href={aboutURL}>About</a> this site!</p>
+```
+
+Enabling i18n routing also provides 
+
+
+## Glossary
+
+The current page will use some specific terminology related to internationalization:
+- Locale: a code that represents a language.
+- Localized folder: a folder/directory that is named after a locale.
+
+*/}
+
+
+<Since v="3.5.0" />
+
+Astro's experimental i18n routing allows you to add your multilingual content with support for configuring a default language and URL path convention, computing relative page URLs, and accepting preferred languages provided by your visitor's browser.
+
+This routing API helps you generate, use, and verify the URLs that your multi-language site produces. Check back and update regularly for the latest changes as this API continues to develop!
+
+
+### Configure i18n routing
+
+1. Enable the experimental routing option by adding an `i18n` object to your Astro configuration with a [default location (`defaultLocale`) and a list of all languages to support (`locales`)](#defaultlocale-and-locales):
+
+   ```js title="astro.config.mjs"
+   import { defineConfig } from "astro/config"
+   export default defineConfig({
+     experimental: {
+       i18n: {
+         defaultLocale: "en",
+         locales: ["en", "es", "pt_BR"]
+       }
+     }
+   })
+   ```
+  
+    You can also specify a [`routingStrategy`](#routing-strategy) to customize the URLs for your default language.
+
+
+2. Organize your content folders with localized content by language, including your `defaultLocale`. Your folder names must match the items in `locales` exactly. `src/pages/index.astro` will now automatically redirect to the `index.astro` file of your default language.
+    <FileTree>
+    - src
+      - pages
+        - en
+          - about.astro
+          - index.astro
+        - es
+          - about.astro
+          - index.astro
+        - pt_BR
+          - about.astro
+          - index.astro
+        - index.astro  (redirects to the route created for `/en/index.astro`)
+    </FileTree>
+
+    :::note
+    The localized folders do not need to be at the root of the `/pages/` folder.
+    
+    Create individual `/[lang]/` folders anywhere within `src/pages/` and Astro's [file-based routing](/en/core-concepts/routing/) will create your pages at corresponding URL paths. 
```

Comment:
I think `[lang]` should be changed to `[locale]` instead, because we never mention "lang" before, instead we mention "locale" already, and we use this terminology inside the configuration. Although, **this is my opinion**, you can disregard it  :)

Reply:
yes! we can change them all to `[locale]`! I think that makes perfect sense. Good call.